src/go/intelligence/main.go

```diff
@@ -36,12 +39,55 @@ func run() {
 	}
 }
 
+func runNoOfTimes() {
```

Comment:
Please use full names without shortcuts

Reply:
Changed the name to changesLogging() containing only logging logics.